### PR TITLE
Global logrus configuration

### PR DESCRIPTION
**Pull request by karlseguin**
## What kind of change does this PR introduce?
Refactor logging, potentially fix some bugs depending on your point of view

## What is the current behavior?
There's currently no global log configuration. Setting `GOTRUE_LOG_LEVEL` is only used by a handful of generated logs. This can be seen by changing the value to "error" in hack/test.env, running the tests, and still observing many INFO logs being generated. Furthermore, some logs are currently generated in plain text while others are in JSON.

## What is the new behavior?
`ConfigureLogging` now configures the logger globally. Any subsequent logger created from the root logger will now inherit all of the configured properties.  Also added distinct handling for POP SQL logs by providing a wrapper around logrus and configurable handling of SQL statements.

## Additional context
I'm familiar with logrus, so beware.

### CODE REVIEW

**Review by hf — COMMENTED** [review]
This is really good! Let me test it out, add some tests, and then we'll merge!

File: conf/logging.go [hunk 1]
```diff
@@ -2,59 +2,105 @@ package conf
...
+		case logging.Debug:
+			l.Debug(s)
+		case logging.Info:
+			l.Info(s)
+		case logging.Warn:
+			l.Warn(s)
+		case logging.Error:
+			l.Error(s)
+		}
+	})
```
Comment (hf) [inline conf/logging.go:98]: All of this argument parsing is handled by pop itself: https://github.com/gobuffalo/pop/blob/v5.3.4/logger.go#L56

I feel like this is too busy and adding too much configuration options. I would just add a new env var `GOTRUE_POP_LOG_LEVEL` and set that to equal the Pop levels.

Comment (kangmingtay) [reply conf/logging.go:98]: we can do that in another PR, will merge this in first so our logs are standardised

**Review by kangmingtay — APPROVED** [review]
hey @karlseguin, thanks for the PR, apologies for the delay in reviewing this!

**Pull request merged**